src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1891,10 +1888,6 @@ public void onSavedCertificate() {
         }
     }
 
-    /**
-     * Called from SslValidatorDialog when a new server certificate could not be saved 
-     * when the user requested it.
-     */
     @Override
     public void onFailedSavingCertificate() {

```

Comment:
And this method?

Reply:
yes, this is also an override. Overides should not be removed even if they are not used.
I've updated the issue because of that.

Reply:
Yes, you're right, I didn't see the Override label